lib/elixir/test/elixir/io/ansi/docs_test.exs

```diff
@@ -303,6 +303,12 @@ defmodule IO.ANSI.DocsTest do
            "\e[7mcolumn 1 | and 2\e[0m\na        | b    \none      | two  \n\e[0m"
   end
 
+  test "table with heading alignment" do
+    table = "column 1 | 2 | and three\n-----: | :------: | :-----\n a      |  even   | c\none | odd | three"
+    expected = "\e[7mcolumn 1 |   2   | and three\e[0m\n       a | even  | c        \n     one |  odd  | three    \n\e[0m"
```

Comment:
Can we use heredocs for those strings? You just need to append <> "\e[0m" at the end I think